docs/content/02-tutorials/06-slurm-rest-api.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "f. Slurm REST API 🌀"
+weight = 26
++++
+
+Enable the Slurm REST API. Requires Slurm Accounting.
+
+## Step 1 - Setup Slurm Accounting
+
+Slurm Accounting is required to enable the Slurm REST API. Follow the [instructions](https://pcluster.cloud/02-tutorials/02-slurm-accounting.html) to enable Slurm Accounting but **do not begin cluster creation** after completing Step 4.
+
+## Step 2 - Create a Security Group to allow inbound HTTPS traffic
+
+By default, your cluster will not be able to accept incoming HTTPS requests to the REST API. You will need to [create a security group](https://console.aws.amazon.com/ec2/v2/home?#CreateSecurityGroup:) to change this.
+
+1. Under `Security group name`, enter "Slurm REST API" (or another name of your choosing)
+2. Ensure `VPC` matches the cluster's VPC
+3. Delete any rules that may have been automatically generated
+4. Add an inbound rule and select `HTTPS` under `Type` and `Anywhere-IPv4` under `Destination`
```

Comment:
**issue**: are we okay to have the HeadNode 443 port opened to the world?

Reply:
We've discussed this a bit but once we get past testing this manually, we should be able instruct the user to set the destination to pcm (assuming we don't automate the security group).